app/helpers/sufia/sufia_helper_behavior.rb

```diff
@@ -121,13 +121,18 @@ def index_field_link(options)
       safe_join(values.map { |item| link_to_field(name, item, item) }, ", ".html_safe)
     end
 
-    # *Sometimes* a Blacklight helper_method
-    # @param text [String,Hash] string to format and escape or a hash as per helper_method
+    # Uses Rails auto_link to add links to fields
+    #
+    # @param [String,Hash] text string to format and escape or a hash as per helper_method
```

Comment:
It's unclear. In the options example, it's the other way around:
http://www.rubydoc.info/gems/yard/file/docs/Tags.md#option